onnx/optimizer/passes/fuse_consecutive_squeezes.h

```diff
@@ -0,0 +1,55 @@
+// ATTENTION: The code in this file is highly EXPERIMENTAL.
+// Adventurous users should note that the APIs will probably change.
+
+#pragma once
+
```

Comment:
Add some comments (pseudo code) to describe the behaviors of the pass: 1) when to fuse squeezes, 2) when to remove the first squeeze